tests/RegistrationForm.test.jsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react'
 import RegistrationForm from '../src/components/RegistrationForm'
-import { render, screen } from '@testing-library/react'
-import { it, expect, describe } from 'vitest'
+import { render, screen, fireEvent, waitFor } from '@testing-library/react'
+import { it, expect, describe, vi } from 'vitest'
+import axios from 'axios'
+import MockAdapter from 'axios-mock-adapter'
```

Comment:
Not sure if you are using it or not, but your mock is probably perfect. Don't think you need this, not the line in the `package.json`.